src/display/annotation_layer.js

```diff
@@ -76,6 +76,15 @@ AnnotationElementFactory.prototype =
         switch (fieldType) {
           case 'Tx':
             return new TextWidgetAnnotationElement(parameters);
+          case 'Btn':
+            if (!parameters.data.pushbutton) {
```

Comment:
Perhaps not really. I'm also fine with a comment such as `// Push buttons are not implemented yet` so that we at least know in the code that something is still missing here.